pom.xml

```diff
@@ -208,7 +208,7 @@
         <dependency>
             <groupId>au.com.dius</groupId>
             <artifactId>pact-jvm-provider-junit_2.11</artifactId>
-            <version>3.3.1</version>
+            <version>3.3.7</version>
```

Comment:
I can see there is a new `3.3.8` release (https://github.com/DiUS/pact-jvm/releases), should we update it?